Python/Product/PythonTools/PythonTools/WorkspaceInfoBarManager.cs

```diff
@@ -128,10 +137,19 @@ class WorkspaceInfoBarManager : IVsRunningDocTableEvents, IDisposable {
                 _condaEnvCreateInfoBar.CheckAsync(),
                 _virtualEnvCreateInfoBar.CheckAsync(),
                 _packageInstallInfoBar.CheckAsync(),
-                _testFrameworkInfoBar.CheckAsync()
+                _testFrameworkInfoBar.CheckAsync(),
+                _pythonVersionNotSupportedInfoBar.CheckAsync()
             );
         }
 
+        private void TriggerPythonNotSupportedInforBar(object sender, EventArgs e) {
+            TriggerPythonNotSupportedInforBarAsync().HandleAllExceptions(_serviceProvider, GetType()).DoNotWait();
+        }
+
+        private async Task TriggerPythonNotSupportedInforBarAsync() {
+            await _pythonVersionNotSupportedInfoBar.CheckAsync();
```

Comment:
async and await are unnecessary overhead here, just `return _pythonVersionNotSupportedInfoBar.CheckAsync();`